index.js

```diff
@@ -0,0 +1,32 @@
+function toggleMenu() {
```

Comment:
- [ ] It is best practice to use an ES6 arrow function so you can replace the first line `function toggleMenu() {` with `const toggleMenu = () => {` and it would still work as you have called it. Kindly apply it to all your functions.